clu/src/lib.rs

```diff
@@ -185,13 +185,21 @@ fn linux_init(config: Arc<RwLock<settings::RitaSettingsStruct>>) -> Result<(), E
                     .to_public_key()
                     .expect("Failed to derive address from Eth key!")
             );
+
+            payment_settings.eth_address = existing_eth_private_key
+                .to_public_key()
+                .expect("Failed to convert Private Key to address!");
```

Comment:
Side notes: this returns Result<> so why not let `linux_init` caller do the .expect?